sonic_platform_base/sonic_xcvr/api/public/c_cmis.py

```diff
@@ -388,363 +388,47 @@ def get_transceiver_bulk_status(self):
-        cfolowalarm_flag             = BOOLEAN                          ; cfo low alarm flag
-        cfohighwarning_flag          = BOOLEAN                          ; cfo high warning flag
-        cfolowwarning_flag           = BOOLEAN                          ; cfo low warning flag
-        txcurrpowerhighalarm_flag    = BOOLEAN                          ; txcurrpower high alarm flag
-        txcurrpowerlowalarm_flag     = BOOLEAN                          ; txcurrpower low alarm flag
-        txcurrpowerhighwarning_flag  = BOOLEAN                          ; txcurrpower high warning flag
-        txcurrpowerlowwarning_flag   = BOOLEAN                          ; txcurrpower low warning flag
-        rxtotpowerhighalarm_flag     = BOOLEAN                          ; rxtotpower high alarm flag
-        rxtotpowerlowalarm_flag      = BOOLEAN                          ; rxtotpower low alarm flag
-        rxtotpowerhighwarning_flag   = BOOLEAN                          ; rxtotpower high warning flag
-        rxtotpowerlowwarning_flag    = BOOLEAN                          ; rxtotpower low warning flag
-        rxsigpowerhighalarm_flag     = BOOLEAN                          ; rxsigpower high alarm flag
-        rxsigpowerlowalarm_flag      = BOOLEAN                          ; rxsigpower low alarm flag
-        rxsigpowerhighwarning_flag   = BOOLEAN                          ; rxsigpower high warning flag
-        rxsigpowerlowwarning_flag    = BOOLEAN                          ; rxsigpower low warning flag
-        ================================================================================
+           Dictionary
         """
         trans_status = super(CCmisApi,self).get_transceiver_status()
         trans_status['tuning_in_progress'] = self.get_tuning_in_progress()
         trans_status['wavelength_unlock_status'] = self.get_wavelength_unlocked()
-        laser_tuning_summary = self.get_laser_tuning_summary()
-        trans_status['target_output_power_oor'] = 'TargetOutputPowerOOR' in laser_tuning_summary
-        trans_status['fine_tuning_oor'] = 'FineTuningOutOfRange' in laser_tuning_summary
-        trans_status['tuning_not_accepted'] = 'TuningNotAccepted' in laser_tuning_summary
-        trans_status['invalid_channel_num'] = 'InvalidChannel' in laser_tuning_summary
-        trans_status['tuning_complete'] = 'TuningComplete' in laser_tuning_summary
-
-        for vdm_key, trans_status_key_prefix in C_CMIS_DELTA_VDM_KEY_TO_DB_PREFIX_KEY_MAP.items():
-            for i in range(5, 9):
-                trans_status_key = trans_status_key_prefix + VDM_SUBTYPE_IDX_MAP[i]
-                self._update_dict_if_vdm_key_exists(trans_status, trans_status_key, vdm_key, i)
 
         return trans_status
 
+    def get_transceiver_status_flags(self):
+        """
+        Retrieves transceiver status flags of this SFP
+        Returns:
+            Dictionary
+        """
+        status_flags_dict = super().get_transceiver_status_flags()
```

Comment:
@mihirpat1 Add comment in code what this status means

Reply:
@prgeor Addressed this now